src/components/LangSwitcher/index.tsx

```diff
@@ -18,30 +18,35 @@ import LangSwitcherContext, {
 } from "./LangSwitcherContext";
 import { getCodeBlocks } from "./utils";
 
+// biome-ignore lint/suspicious/noExplicitAny: <explanation>
```

Comment:
theres's a lot of rough edges in this file (and the sibling `utils` file), i opted to simply fix the typescript errors and `// biome-ignore` the rest (mostly `:any`)